internal/ingest/upload.go

```diff
@@ -92,11 +101,22 @@ func (w *goaWrapper) initSIP(
 	extension string,
 	wType enums.WorkflowType,
 ) error {
+	user, err := w.findOrCreateUser(ctx, w.perSvc)
```

Comment:
No need to pass `w.perSvc`, the method is part of `w` and should have access to it directly.